Cargo.toml

```diff
@@ -1,21 +1,44 @@
 [workspace]
-members = []
+resolver = "2"
+members = ["engine/bin/*", "engine/lib/*"]
 
-[package]
-name = "ltheory"
-authors = ["Limit Theory Redux Team"]
+[workspace.package]
 version = "0.1.0"
-publish = false
+authors = ["Limit Theory Redux Team"]
 edition = "2021"
-autobins = false
+license = "Unlicense"
+# TODO: do we really need these 2 fields?
+# publish = false
+# autobins = false
+
+[workspace.dependencies]
+# workspace bin crates
+ltr = { path = "engine/bin/ltr" }
 
-[[bin]]
-path = "src/Main.rs"
-name = "ltr"
+# workspace lib crates
+phx = { path = "engine/lib/phx" }
+fmod-sys = { path = "engine/lib/fmod-sys" }
+luajit-ffi-gen = { path = "engine/lib/luajit-ffi-gen" }
 
-[dependencies]
-libc = "0.2"
-phx = { path = "libphx" }
+# release crates
+chrono = { version = "0.4.24", default-features = false }
```

Comment:
Do we really need to use the verbose syntax everywhere? I think it makes it look harder to read.

Or maybe a better question is: why are we removing the default features for everything?

Reply:
I thought I put an explanation somewhere but it looks like I didn't.
In short - to prevent a problem when the author of the crate changes default features in the new version of the crate. Plus we specify only necessary features decreasing binary size and compilation time

Reply:
Understood, thanks. I still think this is kind of verbose and I'd be interested to know if there's a measured decrease in the binary size and compilation time, to see if the cost of this (a more verbose Cargo.toml) is worth it.